extra/1-currency-conversion.js

```diff
@@ -15,7 +16,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  return ((price - price / 100) * 5.7).toFixed(2);
```

Comment:
Saman jan! you didn't use 99% of price. you can use something same as this:
(price * 99) / 100